pkgs/stdenv/custom/default.nix

```diff
@@ -15,7 +15,10 @@ in bootStages ++ [
 
   # Additional stage, built using custom stdenv
   (vanillaPackages: {
-    inherit system platform crossSystem config;
+    buildPlatform = localSystem;
+    hostPlatform = localSystem;
+    targetPlatform = localSystem;
```

Comment:
It took me a second, but this is right. `crossSystem` is asserted to be null, and the `*Platforms` must always be non-null.